### PR TITLE
sway: Install the wallpapers by default and use them on NixOS

### PR DESCRIPTION
**Pull request by primeos**
Size impact:
$ du -sh $(nix-build -A sway-unwrapped)
5.9M    /nix/store/d07zcpybgpdj8xhn9n4qb04lgm453ia1-sway-unwrapped-1.6
nix@sessel:/var/tmp/michael/nixpkgs$ du -sh $(nix-build -A sway-unwrapped)/share/backgrounds
4.9M    /nix/store/d07zcpybgpdj8xhn9n4qb04lgm453ia1-sway-unwrapped-1.6/share/backgrounds

Screenshot:
sway_exit

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/window-managers/sway/default.nix [hunk 1]
```diff
@@ -44,7 +47,6 @@ stdenv.mkDerivation rec {
   ];
 
   mesonFlags = [
-    "-Ddefault-wallpaper=false"
```
Comment (primeos) [inline pkgs/applications/window-managers/sway/default.nix:47]: I also considered `"-Ddefault-wallpaper=${boolToString isNixOS}"` but I think the best approach might be to add a `wallpaper` output (and add that to the NixOS module - which makes it still difficult to override but 5 MB are hopefully not that bad - or we keep it as it is).

Comment (primeos) [reply pkgs/applications/window-managers/sway/default.nix:47]: FWIW Arch installs them by default so I guess this should actually be fine: https://archlinux.org/packages/community/x86_64/sway/

**Comment by grahamc** [comment]
It seems to me we may want to instead default it to using NixOS themed wallpapers?

**Comment by primeos** [comment]
@grahamc that's a good question. Personally I prefer sticking to the upstream defaults but if we already default to NixOS themed wallpapers for other compositors/DEs/WMs (e.g. Gnome/KDE) then it might be best to do the same for Sway.

**Comment by Synthetica9** [comment]
I somehow feel it'd be pretty weird if the non-NixOS install came with custom wallpapers... I'd be fine with a Nix wallpaper on NixOS tho

**Comment by Synthetica9** [comment]
Sure thing!

**Pull request merged**